UNRELEASED.md

```diff
@@ -4,6 +4,7 @@
 
 ### Enhancements
 
+- Added `cursor: pointer` prop to `Choice` ([#2491](https://github.com/Shopify/polaris-react/pull/2491))
```

Comment:
```suggestion
- Added `cursor: pointer` to `Choice` ([#2491](https://github.com/Shopify/polaris-react/pull/2491))
```

Reply:
or Added `cursor: pointer` CSS property to `Choice`.

Just because prop might sound like a component prop.